test/Sentry.Serilog.Tests/AspNetCoreIntegrationTests.cs

```diff
@@ -0,0 +1,37 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Hosting;
+using Sentry.Protocol;
+using Sentry.Testing;
+using Xunit;
+
+namespace Sentry.Serilog.Tests
+{
+    [Collection(nameof(SentrySdkCollection))]
+    public class AspNetCoreIntegrationTests : AspNetSentrySdkTestFixture
+    {
+
+#if NET5_0_OR_GREATER || NETCOREAPP3_1_OR_GREATER
```

Comment:
Should include NET5:

```suggestion
#if NETCOREAPP3_1_OR_GREATER
```